BULLETIN.md

```diff
@@ -15,10 +15,43 @@ Highlights and notable changes in this release:
 Auto-GPT now has support for plugins! With plugins, you can extend Auto-GPT's abilities,
```

Comment:
Do we need those 2 paragraphs from 0.3.0? Maybe we just move the ones from bellow here.